src/dialogs/config-flow/dialog-data-entry-flow.ts

```diff
@@ -462,6 +471,49 @@ class DataEntryFlowDialog extends LitElement {
     this._step = undefined;
     await this.updateComplete;
     this._step = _step;
+    if (_step.type === "create_entry" && _step.next_flow) {
+      // skip device rename if there is a chained flow
+      if (_step.next_flow[0] === "config_flow") {
+        this._step = undefined;
+        this._handler = undefined;
+        if (this._unsubDataEntryFlowProgress) {
+          this._unsubDataEntryFlowProgress();
+          this._unsubDataEntryFlowProgress = undefined;
+        }
+        this.showDialog({
```

Comment:
This is a generic component for every flow, we dont know if the previous flow was a `config_flow`, we should just call `showConfigFlow` I think?

Reply:
I was afraid of circular import but it may not be a problem